build.gradle

```diff
@@ -62,7 +62,7 @@ buildscript {
     classpath deps.plugins.android
     classpath 'com.github.ben-manes:gradle-versions-plugin:0.25.0'
     classpath 'org.jetbrains.dokka:dokka-gradle-plugin:0.9.18'
-    classpath 'app.cash.paparazzi:paparazzi-gradle-plugin:1.0.0-square08'
+    //classpath 'app.cash.paparazzi:paparazzi-gradle-plugin:1.0.0-square08'
```

Comment:
what’s going on here?

Reply:
this will let it get past the gradle resolution step, i think waiting to merge this whole thing until the build will go green would be good